R/collect.R

```diff
@@ -368,51 +368,75 @@ collector <- function(x, coll_col = ".predictions") {
 #' @export
 #' @keywords internal
 #' @rdname empty_ellipses
-estimate_tune_results <- function(x, ...) {
+# Get relationship between the parameter values, .config, and (potentially) .iter
+config_key_from_metrics <- function(x) {
```

Comment:
Not sure what the rules are for when internal-ish functions get a period prefix—is this one of those cases?

Reply:
Good point. I'll add a .